src/app/my-families/my-families.component.html

```diff
@@ -14,6 +15,20 @@ <H4 *ngIf="user.escortedHelperName">{{settings.lang.escoring}} {{user.escortedHe
         <mat-icon>refresh</mat-icon>
     </button>
 </div>
+
+<div *ngIf="settings.isSytemForMlt()" style="display:flex;justify-content: center;">
+    <button *ngIf="!this.showQRCode" mat-raised-button (click)="this.showQRCode=true" color="primary">
```

Comment:
לא חסרה אפשרות להסתיר את הQR?
אפשר לעשות משהו כמו:
this.showQRCode = !this.showQRCode

ואם אנחנו כבר כאן - לא צריך את הthis
בHTML הוא בא מעצמו